### PR TITLE
Fifth post

### PR DESCRIPTION
**Pull request by Rohitesh-Kumar-Jain**
I am attaching the blog post for the fifth meeting

Screen Shot 2021-07-12 at 16 54 41-fullpage

### CODE REVIEW

File: jekyll/_posts/2021-07-06-gsod-2021-fifth-meeting.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: GSoD'21 Fifth Meeting
+author: rohitesh-kumar-jain
+categories: gsod
+tags:
+  - google
+  - gsod
+  - event
+  - gsod
```
Comment (lijiarui) [inline jekyll/_posts/2021-07-06-gsod-2021-fifth-meeting.md]: There are two `gsod` tag here, I suggest change `gsod` to `gsod-2021`

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2021-07-06-gsod-2021-fifth-meeting.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+image: /assets/2021/07-gsod-2021-fifth-meeting/gsod-fifth-meeting.webp
+---
+
+## Google Season of Docs'21 fifth meeting
+
+### Meeting Summary
+
+It was the fifth meeting of the GSoD’21 participants. Firstly every participant gave one's weekly summary and informed the community about their plans for the upcoming week, volunteers and tech writers discussed while tech writers were giving their week summaries. Later Simin shared some information regarding the Wechaty follow-ups, and at last, we took a lovely group photo.
+
+{% include iframe.html src="https://youtu.be/WtHPGXxHb5M" %}
```
Comment (lijiarui) [inline jekyll/_posts/2021-07-06-gsod-2021-fifth-meeting.md:22]: When I open this youtube link, I didn't found the progress bar:

image


I suggest you add chapters to a progress bar on youtube by adding `00:00`, see: https://support.google.com/youtube/answer/9884579?hl=en

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2021-07-06-gsod-2021-fifth-meeting.md (hunk 2 above)
Comment (Rohitesh-Kumar-Jain) [reply jekyll/_posts/2021-07-06-gsod-2021-fifth-meeting.md:22]: thanks for the suggestion I have added them

**Review by lijiarui — APPROVED** [review]
Good post! Thanks!

**Review by huan — APPROVED** [review]
Thank you very much for helping the community for organizing the meetings, and making them memorable!

**Pull request merged**